droidlet/dashboard/web/src/components/LiveObjects.js

```diff
@@ -38,8 +38,9 @@ class LiveObjects extends React.Component {
     this.onNextFrame = this.onNextFrame.bind(this);
 
     this.initialState = {
-      height: props.height,
-      width: props.width,
+      height: 640, //  props.height,
```

Comment:
is this intentional? the height width will be different for our habitat setup.

Reply:
good catch, this wasn't intentional, so reverted these changes (which are not needed)